### ISSUE TITLE
Make Pressure Management configurable for each extruder

### ISSUE THREAD

**Issue by curiouspl2** [alexrj/Slic3r#2457]
i see no sense in pressure advance and bridge flow ratio applying to ALL extruders , and being set in 'advanced' tabs. 

imho they belong to 'extruder settings'  , or even filament settings .

this applies especially for setups where one extruder is normal (i.e. perimeter and generally 'main' extruder) and another is bowden (i.e. infill and generally 'low resolution' extruder)

**Comment by lordofhyphens** [comment]
I agree that this should be pushed down to the individual extruders, but note that it will complicate the application of the advance logic.
